ShiftPlan.Blazor.Commons.Tests/EmployeesServiceTests.cs

```diff
@@ -0,0 +1,40 @@
+using FluentAssertions;
+using Moq;
+using ShiftPlan.Blazor.Commons.Clients;
+using ShiftPlan.Blazor.Commons.Models;
+using ShiftPlan.Blazor.Commons.Services;
+
+namespace ShiftPlan.Blazor.Commons.Tests
+{
+	public class EmployeesServiceTests
+	{
+		private readonly EmployeesService _sut;
+		private readonly Mock<IEmployeesClient> _mockEmployeesClient = new Mock<IEmployeesClient>();
```

Comment:
I think `NSubstitute` is easier to read, but we can keep `Mock` if you prefer it 😉

Reply:
just example 

```suggestion
		private readonly IEmployeesClient _mockEmployeesClient = NSubstitute.Subsitute.For<IEmployeeClient>();
```

and in constructor you just use it
```cs
		public EmployeesServiceTests()
		{
			_sut = new EmployeesService(_mockEmployeesClient);
		}
```